version.py

```diff
@@ -45,4 +45,4 @@
 #   historical commits sharing the same major version, in which case a
 #   minor version bump suffices.
 
-PROVISION_VERSION = "135.1"
+PROVISION_VERSION = "136"
```

Comment:
Why does this commit change `PROVISION_VERSION`?  I don't see any changes in dependencies.

Reply:
For migrations.

Reply:
We generally don't update PROVISION_VERSION for migrations; the tools that need to run migrations know to do so automatically.  We only need it for dependency changes that might cause unpredictable issues like "the linter or tests randomly fail with this weird error message" if you're running with a different version.